tracee-ebpf/tracee/containers.go

```diff
@@ -216,14 +217,31 @@ check:
 }
 
 func (c *Containers) CgroupRemove(cgroupId uint64) {
-	delete(c.cgroups, uint32(cgroupId))
+	now := time.Now()
+	// prune containers that have been removed more than 5 seconds ago
+	var deleted []uint64
+	for _, id := range c.deleted {
```

Comment:
I believe its okay to cleanup only when the next event happens (for our use case, which will always contain containers being created/delete). The other idea would be to have a parallel routine doing this check (which I don't think its worth as we're probably solving this with ring buffer approach).

LGTM